WorkflowUI/Sources/Screen/AdaptedEnvironmentScreen.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  File.swift
```

Comment:
A real file's file

Reply:
Ugh, I hate how SPM just makes a file called "File". Removing this and adding the boilerplate.